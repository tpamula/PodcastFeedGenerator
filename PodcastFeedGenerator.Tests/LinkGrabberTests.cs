using ApprovalTests;
using ApprovalTests.Namers;
using ApprovalTests.Reporters;
using PodcastFeedGenerator.Models;
using System.Collections.Generic;
using System.IO;
using Xunit;

namespace PodcastFeedGenerator.Tests
{
    [UseReporter(typeof(DiffReporter))]
    [UseApprovalSubdirectory("Approvals")]
    public class LinkGrabberTests
    {
        private readonly string _testFilePath = @"..\..\Data\IE-2015-04-17.html";

        [Fact]
        private void should_grab_links()
        {
            var input = File.ReadAllText(_testFilePath);

            var linkGrabber = new LinkGrabber();
            var result = linkGrabber.Grab(input);

            Approvals.VerifyAll(result, "[name, link]");
        }
    }
}