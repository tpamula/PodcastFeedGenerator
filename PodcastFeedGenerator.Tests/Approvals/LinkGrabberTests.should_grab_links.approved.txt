[name, link][0] = [Informator ekonomiczny 16.04.2015 09:31, http://static.polskieradio.pl/80dfaeef-9b79-461d-af17-e048595c88e9.mp3]
[name, link][1] = [Informator ekonomiczny 14.04.2015 09:29, http://static.polskieradio.pl/79e5e80c-3d46-47d5-b813-bf6de73beabb.mp3]
[name, link][2] = [Informator ekonomiczny 13.04.2015 09:30, http://static.polskieradio.pl/769f4a3f-839a-4c78-95e4-3d76121dbd09.mp3]
[name, link][3] = [Informator ekonomiczny 09.04.2015 09:29, http://static.polskieradio.pl/7862ae02-2033-4043-ab19-a42f8cbbe0ba.mp3]
[name, link][4] = [Informator ekonomiczny 08.04.2015 09:30, http://static.polskieradio.pl/6bd44387-74ae-4289-b671-1c736a401554.mp3]
[name, link][5] = [Informator ekonomiczny 07.04.2015 09:29, http://static.polskieradio.pl/c3d34b61-ad87-4967-b2e0-d67a6e93df44.mp3]
[name, link][6] = [Informator ekonomiczny 01.04.2015 09:29, http://static.polskieradio.pl/1ec22856-f791-4dbb-b096-2e5da7ec7f2c.mp3]
[name, link][7] = [Informator ekonomiczny 31.03.2015 09:31, http://static.polskieradio.pl/19284cb5-d7e1-4806-a8d3-1c60adc63421.mp3]
[name, link][8] = [Informator ekonomiczny 30.03.2015 09:30, http://static.polskieradio.pl/6a0be5c4-a795-4da6-a798-bbb9ef23f9b6.mp3]
[name, link][9] = [Informator ekonomiczny 26.03.2015 09:30, http://static.polskieradio.pl/04441cf7-dece-4719-98b0-cb407b09aa45.mp3]
[name, link][10] = [Informator ekonomiczny 25.03.2015 09:31, http://static.polskieradio.pl/8fb3c0e2-dd01-496b-90d2-f2ea8e9b5eb7.mp3]
[name, link][11] = [Informator ekonomiczny 24.03.2015 09:30, http://static.polskieradio.pl/7b7b1e5d-5e3f-415e-8cee-c5a47a1c781c.mp3]
[name, link][12] = [Informator ekonomiczny 18.03.2015 09:30, http://static.polskieradio.pl/59720746-b5ed-435d-9d60-095b8afd3da5.mp3]
[name, link][13] = [Informator ekonomiczny 16.03.2015 09:30, http://static.polskieradio.pl/d2e2fda2-9e10-4bb6-8ee0-0cd0c5d2bbba.mp3]
[name, link][14] = [Informator ekonomiczny 11.03.2015 09:31, http://static.polskieradio.pl/2a82d371-60f2-4944-a810-2d53f3b65371.mp3]
[name, link][15] = [Informator ekonomiczny 10.03.2015 09:28, http://static.polskieradio.pl/dddc9846-73ec-4eee-b2e5-07386289d3f7.mp3]
[name, link][16] = [Informator ekonomiczny 09.03.2015 09:29, http://static.polskieradio.pl/77561355-d63d-431a-9d18-24598c5725b8.mp3]
[name, link][17] = [Informator ekonomiczny 04.03.2015 09:30, http://static.polskieradio.pl/04b4c484-e1a2-493b-84c6-df0e8b7878aa.mp3]
[name, link][18] = [Informator ekonomiczny 03.03.2015 09:29, http://static.polskieradio.pl/477ed763-348b-46a4-b4d5-f30025845c65.mp3]
[name, link][19] = [Informator ekonomiczny 02.03.2015 09:30, http://static.polskieradio.pl/2daa871d-a7a4-46c7-ab66-ebfa9d6e2d69.mp3]
[name, link][20] = [Informator ekonomiczny 26.02.2015 09:29, http://static.polskieradio.pl/9bc6cfbd-6e94-477b-ac82-9b030977616b.mp3]
[name, link][21] = [Informator ekonomiczny 25.02.2015 09:30, http://static.polskieradio.pl/62983aed-98d8-4509-acc7-17d94d67560f.mp3]
[name, link][22] = [Informator ekonomiczny 24.02.2015 09:30, http://static.polskieradio.pl/89b6e51e-61df-409d-9816-ff3bfd3e215e.mp3]
[name, link][23] = [Informator ekonomiczny 23.02.2015 09:29, http://static.polskieradio.pl/47280e1b-159f-4d96-b665-2985dc1848c2.mp3]
[name, link][24] = [Informator ekonomiczny 19.02.2015 09:30, http://static.polskieradio.pl/42d624a5-bbe5-4430-82a0-9921a7855288.mp3]
[name, link][25] = [Informator ekonomiczny 18.02.2015 09:29, http://static.polskieradio.pl/6e78e3c4-9144-44c6-b247-4265fc6901da.mp3]
[name, link][26] = [Informator ekonomiczny 17.02.2015 09:28, http://static.polskieradio.pl/e169571d-4b33-4705-9ab6-f4410c993169.mp3]
[name, link][27] = [Informator ekonomiczny 16.02.2015 09:30, http://static.polskieradio.pl/715193ef-7fa6-44fc-b716-0f35f50b026f.mp3]
[name, link][28] = [Informator ekonomiczny 12.02.2015 09:29, http://static.polskieradio.pl/4879f12e-fa5c-46d7-bc5f-6cd5c97fa253.mp3]
[name, link][29] = [Informator ekonomiczny 11.02.2015 09:30, http://static.polskieradio.pl/6e084a6c-70a0-45eb-b28c-2bb794e01faa.mp3]
[name, link][30] = [Informator ekonomiczny 10.02.2015 09:29, http://static.polskieradio.pl/afc248fd-b868-41ba-8bac-2fcef4d3709d.mp3]
[name, link][31] = [Informator ekonomiczny 09.02.2015 09:30, http://static.polskieradio.pl/89c237de-18d2-43ea-b383-a28f3b8ad9a9.mp3]
[name, link][32] = [Informator ekonomiczny 05.02.2015 09:30, http://static.polskieradio.pl/dcfde821-af28-4013-acea-b88b48bca3a0.mp3]
[name, link][33] = [Informator ekonomiczny 04.02.2015 09:30, http://static.polskieradio.pl/f2ea761a-db41-42f1-b379-ef48c77a9e83.mp3]
[name, link][34] = [Informator ekonomiczny 02.02.2015 09:29, http://static.polskieradio.pl/15e45942-3291-4cb4-97a7-d7d108108e60.mp3]
[name, link][35] = [Informator ekonomiczny 29.01.2015 09:29, http://static.polskieradio.pl/aaf5dcc2-d8fa-4433-ac6b-b05e389e44cb.mp3]
[name, link][36] = [Informator ekonomiczny 28.01.2015 09:30, http://static.polskieradio.pl/6fc58ad8-eb60-44b2-b219-2604c487b022.mp3]
[name, link][37] = [Informator ekonomiczny 27.01.2015 09:30, http://static.polskieradio.pl/609b5b0c-a8ad-495e-b120-61ad06b61167.mp3]
[name, link][38] = [Informator ekonomiczny 26.01.2015 09:29, http://static.polskieradio.pl/80292973-03bc-4cd2-83a3-ad8ae7eb3fe5.mp3]
[name, link][39] = [Informator ekonomiczny 22.01.2015 09:29, http://static.polskieradio.pl/1917ea21-f7cf-4c9c-a991-ee47f34c5960.mp3]
[name, link][40] = [Informator ekonomiczny 21.01.2015 09:29, http://static.polskieradio.pl/5f17699e-0666-4584-b925-606eff15cc87.mp3]
[name, link][41] = [Informator ekonomiczny 20.01.2015 09:30, http://static.polskieradio.pl/9ebeb3e9-06ef-4c17-9461-a2ab3be3fbe4.mp3]
[name, link][42] = [Informator ekonomiczny 19.01.2015 09:30, http://static.polskieradio.pl/c1d9c0e6-8df2-4210-adb8-4c441872640c.mp3]
[name, link][43] = [Informator ekonomiczny 15.01.2015 09:31, http://static.polskieradio.pl/32bb638b-4b53-4022-8ab3-c1359c4d4e97.mp3]
[name, link][44] = [Informator ekonomiczny 14.01.2015 09:30, http://static.polskieradio.pl/82acef31-ace3-4900-9360-cf59d95bdff6.mp3]
[name, link][45] = [Informator ekonomiczny 12.01.2015 09:29, http://static.polskieradio.pl/e6ef672e-da15-443c-a961-a372d3091984.mp3]
[name, link][46] = [Informator ekonomiczny 08.01.2015 09:31, http://static.polskieradio.pl/61fd9227-befb-42e4-9da7-d2fe49887636.mp3]
[name, link][47] = [Informator ekonomiczny 07.01.2015 09:30, http://static.polskieradio.pl/56f2eeb2-c40b-420c-834a-266d4cfa5248.mp3]
[name, link][48] = [Informator ekonomiczny 05.01.2015 09:29, http://static.polskieradio.pl/c7d4f4bf-93a2-4599-b3bb-a8eac79e931f.mp3]
[name, link][49] = [Informator ekonomiczny 31.12.2014 09:30, http://static.polskieradio.pl/acabcf20-521c-4459-86a6-c7b19334ab25.mp3]
